com.github.stakira.OpenUtau.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.stakira.OpenUtau
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: OpenUtau
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home #OpenUtau doesn't use portals. https://github.com/stakira/OpenUtau/issues/548
+  - --socket=pulseaudio
+modules:
+
+  - name: OpenUtau
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/
+      - mv * /app/bin/
+      - chmod +x /app/bin/OpenUtau
+
+    sources:
+      - type: archive #I'd like to build from source, but it doesn't seem to work properly and I can't figure out how to make it build for both aarch64 and x86/
```

Comment:
you don't need to do anything specific to build on aarch64. Unless your build system is broken.

Reply:
I mean **on flathub**

Reply:
OpenUtau is built with C#, doesn't dotnet hard-code the arch in the build commands?

Reply:
Also, I encountered a dependency hell when I tried building from source, no clue on how that happened, but it did.

Reply:
I don't know what it does, but you can set in the manifest arch dependent build-options and others. `$FLATPAK_ARCH` is set to the right value. Also we do not cross compile either, so if it pick based on the host architecture, then it should be fine.